src/cluster_slot_stats.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Valkey Contributors.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD 3-Clause
+ */
+
+#include "server.h"
+#include "cluster.h"
+
+void clusterSlotStatReset(int slot);
+void clusterSlotStatsReset(void);
+void clusterSlotStatsAddCpuDuration(int slot, long duration);
```

Comment:
The API is not clear if we are adding `msec` or `usec` ?

Shall we introduce a type via `typedef long usec` or maybe add documentation around the API and rename the `duration_usec`?